### ISSUE TITLE
The plugin is not available from Gradle Plugin Portal

### ISSUE THREAD

**Issue by RichardOsz** [moditect/moditect-gradle-plugin#8]
Hello,

I followed the steps described here 

> https://plugins.gradle.org/plugin/org.moditect.gradleplugin

I am getting the following error when trying to build the project:

```
org.gradle.api.ProjectConfigurationException: A problem occurred configuring root project
...
Caused by: org.gradle.api.internal.artifacts.ivyservice.DefaultLenientConfiguration$ArtifactResolveException: Could not resolve all artifacts for configuration ':classpath'.
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.rethrowFailure(DefaultConfiguration.java:1176)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.access$2100(DefaultConfiguration.java:136)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$ConfigurationArtifactCollection.ensureResolved(DefaultConfiguration.java:1636)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$ConfigurationArtifactCollection.getArtifacts(DefaultConfiguration.java:1608)
	...
	... 108 more
Caused by: org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find com.github.moditect:moditect:master-SNAPSHOT.
Searched in the following locations:
  - https://plugins.gradle.org/m2/com/github/moditect/moditect/master-SNAPSHOT/maven-metadata.xml
  - https://plugins.gradle.org/m2/com/github/moditect/moditect/master-SNAPSHOT/moditect-master-SNAPSHOT.pom
  - https://plugins.gradle.org/m2/com/github/moditect/moditect/master-SNAPSHOT/moditect-master-SNAPSHOT.jar
Required by:
    project : > org.moditect.gradleplugin:org.moditect.gradleplugin.gradle.plugin:1.0.0-beta1 > org.moditect:moditect-gradle-plugin:1.0.0-beta1
	at org.gradle.internal.resolve.result.DefaultBuildableComponentResolveResult.notFound(DefaultBuildableComponentResolveResult.java:42)
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.RepositoryChainComponentMetaDataResolver.resolveModule(RepositoryChainComponentMetaDataResolver.java:108)
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.RepositoryChainComponentMetaDataResolver.resolve(RepositoryChainComponentMetaDataResolver.java:63)
	at org.gradle.api.internal.artifacts.ivyservice.resolveengine.ComponentResolversChain$ComponentMetaDataResolverChain.resolve(ComponentResolversChain.java:95)
	at org.gradle.api.internal.artifacts.ivyservice.clientmodule.ClientModuleResolver.resolve(ClientModuleResolver.java:63)
	...
	... 136 more
```

**Comment by aalmiray** [comment]
The cause of this problem is this line

https://github.com/moditect/moditect-gradle-plugin/blob/master/build.gradle#L34

in combination with Jitpack. The plugin should be configured to pull a released version of the  moditect library, that can be resolved from a standard repository (maven central or jcenter).

**Comment by siordache** [comment]
The current release (1.0.0-beta1) depends on a snapshot of the ModiTect library, which is only available at [JitPack](https://jitpack.io/#moditect/moditect/master-SNAPSHOT). Therefore, you need to add JitPack to the list of repositories used for plugin management. To do this, include the following snippet in your `settings.gradle`:

```
pluginManagement {
    repositories {
        gradlePluginPortal()
        jcenter()
        maven {url "https://jitpack.io"}
    }
}
```

You can use [this file](https://github.com/moditect/moditect-gradle-plugin/blob/27e747640780e1020d138b1ce1e0dfc98f0f6ae4/integrationtest/vert.x/settings.gradle#L1-L7) as a template.

This annoyance will go away with the next ModiTect release (see issue #3).

**Comment by siordache** [comment]
Fixed in 1.0.0-rc1.
